WordPress/Docs/WP/DeprecatedParameterValuesStandard.xml

```diff
@@ -0,0 +1,20 @@
+  
+<documentation title="Deprecated Parameter Values">
+    <standard>
+    <![CDATA[
+    Please refrain from using deprecated WordPress Parameter Values. 
+    ]]>
+    </standard>
+    <code_comparison>
+        <code title="Valid: use of non deprecated values.">
+        <![CDATA[
+bloginfo( $url='' )
+        ]]>
+        </code>
+        <code title="Invalid: use of deprecated values.">
+        <![CDATA[
+bloginfo ( $siteurl ='' ) // Deprecated WP 2.2
+        ]]>
+        </code>
+    </code_comparison>
+</documentation>
```

Comment:
Should be a clear line at the end of the file (End of Line character is not the same as End of File character).